src/components/list/LinkList.tsx

```diff
@@ -3,11 +3,11 @@ import { Link } from "@mui/material";
 import TextItem from "./listItem/TextItem";
 import ExpandableList from "./ExpandableList";
 
-interface PlainLinkBlockProps {
+interface LinkListProps {
   url: string;
 }
 
-const PlainLinkBlock: React.FC<PlainLinkBlockProps> = ({ url }) => {
+const LinkList: React.FC<LinkListProps> = ({ url }) => {
```

Comment:
this name can be confusing with https://en.wikipedia.org/wiki/Linked_list

Reply:
oh, sorry. Let me change another one